### ISSUE TITLE
How can I use python 64bit version for addon. I need use tensorflow framework.

### ISSUE THREAD

**Issue by geniuz** [nvaccess/nvda#11793]

**Comment by leonardder** [comment]
This is currently not possible, NVDA is X86 only.

**Comment by dingpengyu** [comment]
Maybe it's time for NVDA to switch to 64 bit python!
cc 
@michaelDCurran 
and
@leonardder

**Comment by josephsl** [comment]
Hi, not so fast, I think – we cannot forget the common denominator – x86 32-bit for maximum practical portability. Thanks.

**Comment by michaelDCurran** [comment]
There is no advantage for NVDA itself to be 64-bit, and the conversion and packaging changes are not worth the work. Therefore, add-ons must only make use of 32 bit packages and libraries. However, it would be technically possible for an add-on author to write some kind of 64-bit proxy process in Python or another language, and use some kind of IPC to communicate between it and the NVDA add-on code. But as this is very complicated and a very rare use case, NV Access cannot put time towards explaining / designing this kind of thing.

**Comment by feerrenrut** [comment]
A similar issue was encountered during the GSOC 2020 project to add offline image captioning and object detection to NVDA ([see write up here](https://gist.github.com/ShubhamJain7/7986b9056376c9a70e744e0ab3100b85)). This was solved by interfacing with ONNX runtime from C++, after converting the pyTorch models to ONNX format
